lib/class-wp-theme-json.php

```diff
@@ -1197,6 +1194,19 @@ private static function remove_insecure_styles( $input ) {
 		return $output;
 	}
 
+	/**
+	 * Checks that a declaration provided by the user is safe.
+	 *
+	 * @param string $property_name Property name in a CSS declaration, i.e. the `color` in `color: red`.
+	 * @param string $property_value Value in a CSS declaration, i.e. the `red` in `color: red`.
+	 * @return boolean
+	 */
+	private static function validate_style_declaration( $property_name, $property_value ) {
```

Comment:
Can we call this `is_safe_declaration` or something along those lines?

Reply:
Agree, it has no side-effects and returns a boolean so should start with `is_`

I've gone with `is_safe_css_declaration` to disambiguate. There's a lot of different types of declaration in this file, and this method assumes it has already been converted into CSS syntax.